### ISSUE TITLE
Unable to compile racer on both stable and nightly x86_64_pc_msvc builds 

### ISSUE THREAD

**Issue by porites76** [racer-rust/racer#1077]
Hello I am unable to successfully complete the compilation of racer. It gives the following error: 

error[E0046]: not all trait items implemented, missing: `mixed_site`
   --> ~\registry\src\github.com-1ecc6299db9ec823\rustc-ap-syntax-583.0.0\ext\proc_macro_server.rs:657:1
    |
657 | impl server::Span for Rustc<'_> {
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ missing `mixed_site` in implementation
    |
    = note: `mixed_site` from trait: `fn(&mut Self) -> <Self as proc_macro::bridge::server::Types>::Span`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0046`.
error: could not compile `rustc-ap-syntax`.
warning: build failed, waiting for other jobs to finish...

The build details are as follows:
1) nightly-x86_64-pc-windows-msvc unchanged - rustc 1.40.0-nightly (2748a9fd9 2019-10-08)
2) stable-x86_64-pc-windows-msvc unchanged - rustc 1.38.0 (625451e37 2019-09-23)
3) rustc version - 1.40.0-nightly (2748a9fd9 2019-10-08)
4) OS - Windows 10 Home

I use the following command for installation: 
cargo +nightly install racer 

switching default to both nightly and stable doesn't help. Is this a bug or racer isn't supported on windows 10 ?

**Comment by kngwyu** [comment]
Sorry, I'll release the new version soon.

**Comment by zhangxu-ai** [comment]
> Sorry, I'll release the new version soon.

do you release the new version now?

**Comment by havokca** [comment]
Having the same issue while trying to get vs code installed and playing nice with rust.

**Comment by worikgh** [comment]
I have had same problem
2.1.27 compiles tho and passes all tests

**Comment by easye** [comment]
One can go back in time to get a nightly that works, right?  Anybody have a date for which that would work? 

/me experiments

**Comment by easye** [comment]
> One can go back in time to get a nightly that works, right? Anybody have a date for which that would work?
> 

After experimenting, I found that 
```
rustup install nightly-2019-10-01
cargo +nightly-2019-10-01 install racer  
```
will successfully install racer (at least on FreeBSD/macOS).

**Comment by kngwyu** [comment]
Released 2.1.28 now.
